ts/packages/ts/src/commands/ExportCommand.ts

```diff
@@ -1,22 +1,57 @@
 import _ from 'lodash';
 import { Command } from '../Command';
 import { type EvalCommandReturn } from './EvalCommand';
+import { NAME, ConfigError } from '../utils';
 
 export type ExportCommandReturn = {
   [key: string]: string;
 };
 
 export type ExportCommandParameters = {
   pipe: EvalCommandReturn;
+  keys?: (key: string) => string;
 };
 
 export class ExportCommand extends Command<ExportCommandReturn> {
   constructor(public parameters: ExportCommandParameters) {
     super(parameters);
   }
 
+  private mutateKeys(result: { [key: string]: string }): { [key: string]: string } {
+    const { keys } = this.parameters;
+
+    if (!keys) {
+      return result;
+    }
+
+    return _.mapKeys(result, (current, key) => {
+      const errorScope: [string, string][] = [
+        ['ExportCommand', ''],
+        ['ConfigKey', key],
+      ];
+      let mutatedKey = '';
+      let isKeyValid = true;
+      try {
+        mutatedKey = String(keys(key));
+      } catch (error) {
+        isKeyValid = false;
+      }
```

Comment:
I'm not sure why we are masking errors that come from `keys`, unless there's a reason I'm not aware of, I think we should let the user see the error that comes from their own function

Reply:
done